docs/BR.md

```diff
@@ -1286,29 +1286,36 @@ No stipulation.
 
 ### 4.9.7 CRL issuance frequency
 
-CAs MUST generate and publish either:
-- a full and complete; OR
+CRLs must be available via a publicly-accessible HTTP URL (i.e., "published").
+
+Within 24 hours of issuing its first Certificate, CAs MUST generate and publish either:
+- a full and complete CRL; OR
 - partitioned (i.e., "sharded") CRLs, that when aggregated, represent the equivalent of the full and complete CRL.
 
-CRLs must be available via a publicly-accessible HTTP URL.
+CAs issuing Subscriber Certififcates MUST continue to issue CRLs in accordance with the table below:
+
+| __Conditions__ | __Minimum CRL Issuance Frequency__ | __Maximum nextUpdate Period__ | 
+| ----           | -                                  | -                             |
+| The CA has revoked at least one unexpired Certificate since the `thisUpdate` field of the last CRL. | A CRL MUST be generated and published within 24 hours of the `thisUpdate` field of the last CRL. <br><br> A CRL SHOULD be generated and published within 4 hours of the `thisUpdate` field of the current CRL. | The value of the `nextUpdate` field MUST NOT be more than 10 days beyond the value of the `thisUpdate` field. |
+| All other conditions. | A CRL MUST be generated and published within 7 days of the `thisUpdate` field of the last CRL. <br><br> A CRL SHOULD be generated and published within 4 hours of the `thisUpdate` field of the current CRL. | The value of the `nextUpdate` field MUST NOT be more than 10 days beyond the value of the `thisUpdate` field. |
 
-CAs issuing Subscriber Certificates (i.e., Certificates for servers) SHALL:
-- update and reissue CRLs at least once every 24 hours; and
-- the value of the `nextUpdate` field MUST NOT be more than ten (10) days beyond the value of the `thisUpdate` field.
+CAs issuing CA Certificates MUST issue CRLs in accordance with the table below:
+| __Conditions__ | __Minimum CRL Issuance Frequency__ | __Maximum nextUpdate Period__ | 
+| ----           | -                                  | -                             |
+| The CA has revoked at least one unexpired Subordinate CA Certificate since the `thisUpdate` field of the last CRL. | A CRL MUST be generated and published within 24 hours of the `thisUpdate` field of the last CRL. <br><br> A CRL SHOULD be generated and published within 4 hours of the `thisUpdate` field of the current CRL. | The value of the `nextUpdate` field MUST NOT be more than twelve months beyond the value of the `thisUpdate` field. |
```

Comment:
As above, this phrasing is also impossible. If a Root CA issues a 12-month CRL, then 6 months pass, then it needs to revoke one of its Subordinate CA Certificates, it is stuck: it "has revoked" (see comment above) at least one cert since the thisUpdate of the previous CRL, but significantly more than 24 hours have already passed since that previous CRL.

Reply:
Please see https://github.com/ryancdickson/staging/pull/3#discussion_r1186880435.

Reply:
I'll mark this comment as resolved given the "table" presentation format was later abandoned. I included @dzacharo suggested edits on 4.9.7 and think that it reads well.

Any further comments or concerns on the current 4.9.7 text can be discussed in the open comment above.